service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/ErrorConstants.java

```diff
@@ -0,0 +1,23 @@
+package uk.nhs.adaptors.gp2gp.ehr.utils;
+
+public enum ErrorConstants {
+
+    ACK_TIMEOUT("99", "No acknowledgement has been received within ACK timeout limit");
+
+    private final String code;
+    private final String message;
+
+    ErrorConstants(String code, String message) {
+        this.code = code;
+        this.message = message;
+    }
+
+    public String getCode() {
+        return code;
+    }
+
+    public String getMessage() {
+        return message;
+    }
```

Comment:
If we're using Lombok within the project, we _could_ use `@Getter` at class level - but happy for it to stay as-is, your call! 😄